src/backend/src/services/change-requests.services.ts

```diff
@@ -412,16 +426,24 @@ export default class ChangeRequestsService {
     });
 
     const teams = createdCR.wbsElement.workPackage?.project.teams;
-
+    const notifications: { channelId: string; ts: string }[] = [];
     if (teams && teams.length > 0) {
       teams.forEach(async (team) => {
         const slackMsg =
           `${submitter.firstName} ${submitter.lastName} wants to activate ${createdCR.wbsElement.name}` +
           ` in ${createdCR.wbsElement.workPackage?.project.wbsElement.name}`;
-        await sendSlackChangeRequestNotification(team, slackMsg, createdCR.crId);
+        const sentNotifications: { channelId: string; ts: string }[] = await sendSlackChangeRequestNotification(
+          team,
+          slackMsg,
+          createdCR.crId
+        );
+        if (sentNotifications) notifications.push(...sentNotifications);
```

Comment:
Not sure if this is possible because of the possibility of there being multiple teams meaning multiple notifications, where the notifications would be returned in different scopes since it uses teams.forEach, unless you want me to completely refactor `sendSlackChangeRequestNotification`

Reply:
I see what you're saying, I think you good find a way to reduce a lot of this duplicate code would be by abstracting the code in the forEach call into a helper method, with inputs that allow it to be used in each of these 3 cases (cause I know we're sending 3 different messages)